handlers/amazonec2_test.go

```diff
@@ -50,7 +50,7 @@ func TestAmazonec2(t *testing.T) {
 func setup(accessKey string, secretKey string, vpcId string, zone string) {
 	// TODO Replace functions during teardown.
 	machine := &client.Machine{
-		Amazonec2Config: client.Amazonec2Config{
+		Amazonec2Config: &client.Amazonec2Config{
```

Comment:
@cjellick I suspect that is because I haven't updated the Godeps with the changes to go-rancher. Was planning on ammending this commit once go-rancher was merged. I'll do it now anyways, that way we can see the build passing